### DISCUSSION TITLE
Preserve groupIDs/crysym/... when generating a .tess-file out of an .tesr-file

### DISCUSSION THREAD

**Discussion by schmid02** [neperfepx/neper#949] [Q&A] [answered]
### Description
I've generated an .tesr-file of an 2-phase EBSD-image containing the phaseIDs in the ```*group```-section and the symmetry ```cubic``` in the ```*crysym```-section. So my .tesr-file looks like the output of this:
```
$ neper -T -n 10 -group "id<=5?1:2" -crysym cubic -format tesr -o ebsd
```
When I build an .tess-file out of it in the following way, the new .tess-file is missing the  ```*group```-section and the crysym is ```triclinic``:
```
$ neper -T -n from_morpho -morpho "tesr:file(ebsd.tesr)"
```

### My Quick-Fix
I fixed that by providing the groupIDs in a data file, which I loaded with ```-group "file(groupIDs.txt)"``` and specifiying the symmetry with ```-crysym cubic```.

### Question
But I wondered, is there's a way to read the groupIDs and crysym directly from the .tesr-file?

**Comment by rquey** [comment, marked as answer]
Fixed in version 4.10.2-8. Thanks for the bug report.
